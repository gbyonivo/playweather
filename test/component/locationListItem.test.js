import React from 'react';
import renderer from 'react-test-renderer';
import LocationListMenu from '../../src/components/locationListMenuItem';

describe('LocationListMenu component', () => {
  it('renders correctly when isLoading is true', () => {
    const actual = renderer.create(
      <LocationListMenu
        location="play"
        selectLocation={() => {}}
      />
    );
    expect(actual).toMatchSnapshot();
  });
});
