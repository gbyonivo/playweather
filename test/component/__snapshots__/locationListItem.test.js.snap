// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LocationListMenu component renders correctly when isLoading is true 1`] = `
<li
  className="locationListMenuItem"
  onClick={[Function]}
>
  play
</li>
`;
